### ISSUE TITLE
Fix a typo in a comment in t.test.test_reflect

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7369]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#7369|
|Type|defect|
|Created|2014-05-18 15:15:02Z|
|Branch|https://github.com/twisted/twisted/tree/comment-typo-reflect-7369|

<details><summary>Searchable metadata</summary>

```
trac-id__7369 7369
type__defect defect
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_comment_typo_reflect_7369 branches/comment-typo-reflect-7369
branch_author__julian julian
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1400426102000000 1400426102000000
changetime__1400431569000000 1400431569000000
version__None None
owner__Julian Julian

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|julian commented|
|-|-|

(In [42660]) Branching to comment-typo-reflect-7369.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @Julian|
|-|-|

Thanks.

The problem fixed in the branch was introduced in r22245, fixing [#11357](https://github.com/twisted/twisted/issues/11357).

Please describe the problem being fixed in the ticket description in more detail in the future.

The change looks correct to me.  Please merge.  Thanks again.

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|julian set status to `closed`|
|-|-|

(In [42662]) Merge comment-typo-reflect-7369: Fix a typo in a namedAny test.

Author: julian
Reviewers: exarkun
Fixes: [#7369](https://github.com/twisted/twisted/issues/7369)

This test means to say that assertIs won't work, since ... etc.
